examples/test_features.py

```diff
@@ -52,9 +52,9 @@ def est_features_run(PATH_RUN) -> None:
 
     to_ref_idx = np.array(df_M1[(df_M1['used'] == 1)].index)
 
-    cortex_idx = np.where(df_M1.ECOG == 1)[0]
-    subcortex_idx = np.array(df_M1[(df_M1["ECOG"] == 0) & (df_M1['used'] == 1)
-                                   & (df_M1['target'] == 0)].index)
+    cortex_idx = np.where(df_M1.type == 'ecog')[0]
+    subcortex_idx = np.array(df_M1[(df_M1["type"] == 'seeg') | (df_M1['type'] == 'dbs')
+                                   & (df_M1['type'] == 'lfp')].index)
```

Comment:
@timonmerk  Shouldn't there be a logical "or" before the "lfp" typecheck? Like this:
`| (df_M1['type'] == 'lfp')`